tests/Doctrine/Tests/DBAL/Functional/Ticket/DBAL2369Test.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Doctrine\Tests\DBAL\Functional\Ticket;
+
+use Doctrine\DBAL\DBALException;
+use Doctrine\DBAL\ParameterType;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\Tests\DbalFunctionalTestCase;
+
+/**
+ * @group DBAL-2369
+ */
+class DBAL2369Test extends DbalFunctionalTestCase
+{
+    /**
+     * @throws DBALException
+     */
+    protected function setUp() : void
+    {
+        parent::setUp();
+
+        if ($this->_conn->getDatabasePlatform()->getName() !== 'mssql') {
+            $this->markTestSkipped('Related to SQLSRV only');
+        }
+
+        /* @var $sm \Doctrine\DBAL\Schema\AbstractSchemaManager */
+        $sm = $this->_conn->getSchemaManager();
+        if ($sm->tablesExist(['dbal2369'])) {
```

Comment:
Please do `$sm->dropAndCreateTable()` instead. The fact that the table exists doesn't mean it has the expected schema.